src/utils/use-refresh-on-focus.ts

```diff
@@ -0,0 +1,23 @@
+import {useIsFocused} from '@react-navigation/native';
+import {differenceInSeconds} from 'date-fns';
+import {useEffect} from 'react';
+
+export function useRefreshOnFocus(
+  tick: Date | undefined,
```

Comment:
Can this be called `lastUpdate` or something? If not I think it can be documented. It is kinda hard by reading just the parameters what `tick` actually means.